### ISSUE TITLE
log a warning if PIDFile uses /var/run

### ISSUE THREAD

**Issue by mbiebl** [systemd/systemd#10657]
Newer versions of systemd-tmpfiles log a warning, if a tmpfiles snippet uses /var/run:
```
[/usr/lib/tmpfiles.d/spice-vdagentd.conf:2] Line references path below legacy directory /var/run/, updating /var/run/spice-vdagentd → /run/spice-vdagentd; please update the tmpfiles.d/ drop-in file accordingly.
```
I think it would be useful to do the same for PIDFile in .service files to gently push maintainers to use /run.
Atm we have quite a few service files using /var/run:
https://codesearch.debian.net/search?q=PIDFile%3D%2Fvar%2Frun

**Comment by mbiebl** [comment]
On second thought, given how many packages (still) use /var/run, maybe make this debug prio.

**Comment by poettering** [comment]
the main reason i added that warning to tmpfiles is that tmpfiles tries to merge lines for the same files and that is quirky if the lines use different prefixes: some use /var/run and others /run...

Now, this reason doesn't really apply to PID files, but you are right, given that we do this for tmpfiles we should do it for all similar cases too i figure.

**Comment by poettering** [comment]
FIx waiting in #10712.
